index.html

```diff
@@ -1595,6 +1573,17 @@ <h3>
         </dl>
       </section>
     </section>
+    <section id="same-origin-check">
```

Comment:
Let's add this section before the obsolete section.

Reply:
bumped it up